### ISSUE TITLE
💅 Print the entire file when using `--stdin-file-path`

### ISSUE THREAD

**Issue by adaliszk** [biomejs/biome#2544]
I wanted to apply linting fixes, including unsafe ones like the tailwind class order. In my environment, I use `dprint` to run formatters, and I figured I would add a simple command with the linting so that biome would be run with all the configurations.
In that setup, it is expected to get the entire formatted output when executing the command. Since there is a way to use the `--stdin-file-path` option, I configured it but learned that unsupported portions of the source file get omitted, thus deleting my code through this method.

### Environment information

```bash
CLI:
  Version:                      1.7.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v20.10.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 false

Linter:
  Recommended:                  true
  All:                          false
  Rules:
Workspace:
  Open Documents:               0
```

### Rule name

tailwind class order

### Playground link

https://codesandbox.io/p/devbox/biome-stout-omits-unsupported-parts-hf2vj3

Run `npm run issue` to simulate the problem. (On Codesandbox, it seems to be hanging, but on a real machine, it does work)

### Expected result

When using the STDIN, the STDOUT should contain all the file contents, even if portions are unsupported.
This allows external tools, like `dprint`, to receive the output and write to the disk.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Oh, that's an astro file. Good to know.

FYI it's a specific issue for partially supported files. JS files aren't affected, for example

**Comment by adaliszk** [comment]
I see, sorry for missing that detail in the report.

**Comment by ematipico** [comment]
This should be fixed in v1.7.3
